lib/tasks/feeds/move.rake

```diff
@@ -0,0 +1,22 @@
+class CloudDataFeed
```

Comment:
this PR should be fine, so we can merge that, and the `CloudDataFeed` will be included
https://github.com/ministryofjustice/hmpps-book-secure-move-api/pull/803

it took me a bit more than expected to write the tests 😭